### PR TITLE
4.4.0: pushing WIP code for phylactery check RPC

### PR DESCRIPTION
**Pull request by jpw1991**

### CODE REVIEW

**Comment by CW-Jesse** [comment]
Gotta remove Phylacteries from the static list OnDestroy or similar.

**Comment by CW-Jesse** [comment]
Would it be better to do checks for fueled Phylacteries when the player is supposed to die instead of constantly updating it? Make them invincible and then kill them if one isn't available?

**Comment by jpw1991** [comment]
> Would it be better to do checks for fueled Phylacteries when the player is supposed to die instead of constantly updating it? Make them invincible and then kill them if one isn't available?

It currently is supposed to work like:

- Every 5 seconds, client asks server if it has a phylactery made by the player that has fuel
- If it does, the `HasPhylacter`

File: ChebsNecromancy/Structures/Phylactery.cs [hunk 1]
```diff
@@ -51,9 +63,60 @@ public static void CreateConfigs(BasePlugin plugin)
                     new ConfigurationManagerAttributes { IsAdminOnly = true }));
         }
 
-        public bool ConsumeFuel()
+        public static void ConfigureRPC()
+        {
+            PhylacteryCheckRPC = NetworkManager.Instance.AddRPC("PhylacteryCheckRPC",
```
Comment (CW-Jesse) [inline ChebsNecromancy/Structures/Phylactery.cs:68]: Configure the server version only on the server, and the client version only on the client.

**Comment by jpw1991** [comment]
@CW-Jesse code review still welcome on this but tested to work on both single player and dedicated server

**Pull request merged**